internal/config/init.go

```diff
@@ -142,6 +144,23 @@ func LoadWithDefault() Conf {
 		c.GitHubURL = os.Getenv(EnvGitHubURL)
 	}
 
+	c.RunnerVersion = "latest"
+	if os.Getenv(EnvRunnerVersion) != "" {
+		ephemeralSupportVersion, err := version.NewVersion("v2.282.0")
+		if err != nil {
+			log.Panicf("failed to parse ephemeral runner version: %+v", err)
+		}
+		inputVersion, err := version.NewVersion(os.Getenv(EnvRunnerVersion))
+		if err != nil {
+			log.Panicf("failed to parse input runner version: %+v", err)
+		}
+		if !ephemeralSupportVersion.GreaterThan(inputVersion) {
+			log.Panicf("RUNNER_VERSION must be greater than %s", ephemeralSupportVersion)
```

Comment:
myshoes also support the old version's less ephemeral mode.
you needn't panic here.

Reply:
I deleted that, but I think need to panic when version is invalid.